pushapkscript/docker.d/init_worker.sh

```diff
@@ -98,6 +98,8 @@ case $COT_PRODUCT in
         test_var_set 'GOOGLE_CREDENTIALS_REFERENCE_BROWSER'
         test_var_set 'GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_AURORA'
         test_var_set 'GOOGLE_CREDENTIALS_FIREFOX_AURORA'
+        test_var_set 'GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_BETA'
+        test_var_set 'GOOGLE_CREDENTIALS_FIREFOX_BETA'
```

Comment:
Also, a few lines below, I believe we are missing a few lines in this section:
```
export GOOGLE_CREDENTIALS_FIREFOX_BETA_PATH=$CONFIG_DIR/beta.p12
echo $GOOGLE_CREDENTIALS_FIREFOX_BETA | base64 -d >        $GOOGLE_CREDENTIALS_FIREFOX_BETA_PATH
```

Reply:
👍 Fixed!